public/app/features/playlist/PlaylistPage.test.tsx

```diff
@@ -14,12 +16,6 @@ jest.mock('@grafana/runtime', () => ({
   }),
 }));
 
-jest.mock('app/core/services/context_srv', () => ({
-  contextSrv: {
-    isEditor: true,
-  },
-}));
```

Comment:
Please put this mock back in. These tests will now modify `contextSrv` globally and can make tests indeterministic.

Reply:
Good point. I've re-added the mock.